### PR TITLE
sol

### PR DESCRIPTION
**Pull request by KrisMur**
[DEMO LINK](https://KrisMur.github.io/react_tabs-with-router/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,48 +1,70 @@
...
-      <h1 className="title">Page not found</h1>
+  return (
+    <div className="App">
+      <nav className="navbar has-text-centered">
+        <NavLink to="/" className="navbar-item">
+          Home
+        </NavLink>
+        <NavLink to="/tabs" className="navbar-item">
+          Tabs
+        </NavLink>
```
Comment (jstmpelowycz) [inline src/App.tsx:28]: review the checklist, p.1

File: src/App.tsx [hunk 2]
```diff
@@ -1,48 +1,70 @@
...
+            </h1>
+          )}
+        />
 
-      <div className="block" data-cy="tab-content">
-        Please select a tab
-      </div>
+        <Route
+          path="tabs"
+          element={<Tabs tabs={tabs} selectedTabId={selectedTab?.id} />}
```
Comment (jstmpelowycz) [inline src/App.tsx:43]: you shouldn't pass tab id, use routes hooks to retrieve what you need

File: src/App.tsx [hunk 3]
```diff
@@ -1,48 +1,70 @@
...
+        >
+          <Route
+            index
+            element={(
+              <TabsPage
+                onSelect={onSelect}
+                selectedTab={selectedTab}
```
Comment (jstmpelowycz) [inline src/App.tsx:50]: with tab – as well

File: src/conponents/TabsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { tabs } from '../api/tabs';
```
Comment (jstmpelowycz) [inline src/conponents/TabsPage.tsx:3]: pass them as props

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**